test/t_validate_model.cc

```diff
@@ -0,0 +1,32 @@
+#include "unit-include/config.h"
+#include "unit-include/util.h"
+#include "unit-include/validate_aes_model.h"
```

Comment:
There’s already an exported JSON file for AES in `unit-data`.

Reply:
Actually, I customized the json with a pair of conflict decode functions (both true under some state). 
Anyway, I switched to the aes_v.json now.